Makefile.toml

```diff
@@ -997,23 +1018,27 @@ echo If you still feel this is not enough, you may copy $(tput setaf 4)risedev$(
 """
 
 [tasks.ci-start]
+private = true
```

Comment:
From my perspective, I heavily rely on `ci-start` locally for a shortcut of `clean` and `dev`. 🥵